docs/how-to-work-on-coding-challenges.md

```diff
@@ -209,13 +209,16 @@ Our goal is to have thousands of 2-minute challenges. These can flow together an
 Here are specific formatting guidelines for challenge text and examples:
 
 - Language keywords go in `<code>` tags. For example, HTML tag names or CSS property names
-- The first instance of a keyword when it's being defined, or general keywords (i.e. "object" or "immutable") go in `<dfn>` tags
+- The first instance of a keyword when it's being defined, or general keywords (e.g. "object" or "immutable") go in `<dfn>` tags
 - References to code parts (i.e. function, method or variable names) should be wrapped in `<code>` tags. See example below:
-- Use <code>parseInt</code> to convert the variable <code>realNumber</code> into an integer.
-- Multi-line code blocks **must be preceded by an empty line**. The next line must start with three backticks followed immediately by one of the [supported languages](https://prismjs.com/#supported-languages). To complete the code block, you must start a newline which only has three backticks and **another empty line**.
-  **Note:** If you are going to use an example code in YAML, use `yaml` instead of `yml` for the language to the right of the backticks.
-
-See example below:
+```md
+Use <code>parseInt</code> to convert the variable <code>realNumber</code> into an integer.
+```
+- References to file names and path directories (e.g. `package.json`, `src/components`) should be wrapped in `<code>` tags.
+- Mention of tools, packages, and trademarks (e.g. _Pug, MongoDB_) should be wrapped in `<em>` tags.
```

Comment:
I don't know why we need `em` tags around any of these things.

Reply:
Would you have them in code tags? Or, plain text?

Reply:
> Would you have them in code tags? Or, plain text?

@Sky020 If we are referencing a package the user must install or use, the use `code` tags.  If you are just referencing a library like React, then nothing should surround it.  The same would go for a tool.  As far as trademarks, I see no reason to surround with anything special.